app/models/loader/types.rb

```diff
@@ -134,8 +134,24 @@ def verify; end
     end
 
     class Host < Record
-      def verify; end
-      def_delegators :@policy_object, :restricted_to
+      def_delegators :@policy_object, :restricted_to, :apikey
+
+      def api_key
```

Comment:
This doesn't actually return an API, but rather, whether they're enabled. So this should probably be named something like `api_key_enabled?` (The '?' at the end is a common Ruby idiom for methods that return a boolean value.

Reply:
Done